<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1491</width>
      <height>1161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="consumptionTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Consumption Graphs</string>
     </attribute>
     <widget class="QPushButton" name="householdButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Households</string>
      </property>
     </widget>
     <widget class="QLabel" name="endDateLabel">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>End Date:</string>
      </property>
     </widget>
     <widget class="matplotlib_widget" name="matplotlib_widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>1111</width>
        <height>721</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="startDateLabel">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start Date:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditOutput">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="maximumDate">
       <date>
        <year>7914</year>
        <month>2</month>
        <day>28</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2012</year>
        <month>10</month>
        <day>12</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1491</width>
        <height>1051</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/powerPlant2.jpg</pixmap>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditInput">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="maximumDate">
       <date>
        <year>7914</year>
        <month>2</month>
        <day>28</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2012</year>
        <month>10</month>
        <day>12</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="graphButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>30</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Click to see the graph</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="blocksButton_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Blocks</string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>graphButton</zorder>
     <zorder>endDateLabel</zorder>
     <zorder>householdButton</zorder>
     <zorder>dateEditOutput</zorder>
     <zorder>dateEditInput</zorder>
     <zorder>matplotlib_widget</zorder>
     <zorder>startDateLabel</zorder>
     <zorder>blocksButton_1</zorder>
    </widget>
    <widget class="QWidget" name="classificationTab">
     <attribute name="title">
      <string>Classification</string>
     </attribute>
     <widget class="QPushButton" name="classificationButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Click to see classification</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1491</width>
        <height>1051</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/powerPlant2.jpg</pixmap>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="figureLabel_1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>960</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="blocksButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Blocks</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>classificationButton</zorder>
     <zorder>figureLabel_1</zorder>
     <zorder>blocksButton_2</zorder>
    </widget>
    <widget class="QWidget" name="newHouseholdTab">
     <attribute name="title">
      <string>Add New Household</string>
     </attribute>
     <widget class="QLineEdit" name="houseID">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string comment="0">House ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="peakTime">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Peak Load Time</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="energy">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>20</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Total Consumed Energy (kW)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showButton">
      <property name="geometry">
       <rect>
        <x>515</x>
        <y>80</y>
        <width>131</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add and Show</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1491</width>
        <height>1051</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/powerPlant2.jpg</pixmap>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="figureLabel_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>960</width>
        <height>720</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="blocksButton_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Blocks</string>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>houseID</zorder>
     <zorder>peakTime</zorder>
     <zorder>energy</zorder>
     <zorder>showButton</zorder>
     <zorder>figureLabel_2</zorder>
     <zorder>blocksButton_3</zorder>
    </widget>
    <widget class="QWidget" name="loadShiftingTab">
     <attribute name="title">
      <string>Load Shifting</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1491</width>
        <height>1051</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/powerPlant2.jpg</pixmap>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="blocksButton_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Blocks</string>
      </property>
     </widget>
     <widget class="QLabel" name="figureLabel_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>544</width>
        <height>408</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="figureLabel_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>230</y>
        <width>544</width>
        <height>408</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="percentageButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Percentages</string>
      </property>
     </widget>
     <widget class="QPushButton" name="shiftingButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>60</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Click to see shifted version</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matplotlib_widget</class>
   <extends>QWidget</extends>
   <header>matplotlib_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
